test/e2e/e2e.go

```diff
@@ -127,36 +127,47 @@ func CreateSubscription(clients *test.Clients, sub *v1alpha1.Subscription, _ log
 	return nil
 }
 
-// WithChannelAndSubscriptionReady creates a Channel and Subscription and waits until both are Ready.
-func WithChannelAndSubscriptionReady(clients *test.Clients, channel *v1alpha1.Channel, sub *v1alpha1.Subscription, logf logging.FormatLogger, cleaner *test.Cleaner) error {
-	if err := CreateChannel(clients, channel, logf, cleaner); err != nil {
-		return err
-	}
-	if err := CreateSubscription(clients, sub, logf, cleaner); err != nil {
-		return err
+// WithChannelsAndSubscriptionsReady creates Channels and Subscriptions and waits until all are Ready.
```

Comment:
Document that `chans` and `subs` are being altered to get the real Channels and Subscriptions.

Reply:
Added the comment.